### ISSUE TITLE
BUG: Panels "visible" attribute doesn't prevent panel rendering

### ISSUE THREAD

**Issue by emyasnikov** [artf/grapesjs#3418]
**Version:**

v0.17.3

**Are you able to reproduce the bug from the demo?**

 - [ ] Yes
 - [x] No

**What is the expected behavior?**

`editor.Panels.addPanel({id: 'test', visible: false})` renders panel despite of `visible` attribute set to false.
It perfectly works after editor has been rendered, so `panel.set('visibility', !panel.get('visibility')` toggles visibility correctly.

Workaround 1 to set `gjs-hidden` class with `visible: false` together didn't work too. This way visibility would be checked later and `gjs-hidden` class would be removed. At least is this the behavior I see.

There was same issue already: [#1602](https://github.com/artf/grapesjs/issues/1602).

Workaround 2 to call `toggleVisibility` function just after panel has been created didn't work too. I think it's because `render()` function in the [PanelView](https://github.com/artf/grapesjs/blob/dev/src/panels/view/PanelView.js#L121) overrides classes. Even to set class manually like it's done in the [toggleVisible](https://github.com/artf/grapesjs/blob/dev/src/panels/view/PanelView.js#L34) function didn't work.

**What is the current behavior?**

Visible attribute is set correctly to `false` in the panel when I get it using `editor.Panels.getPanel('test')` but panel itself is still visible until `panel.set('visible', false)` call after editor has been rendered.

**Are you able to attach screenshots, screencasts or a live demo?**

 - [ ] Yes (attach)
 - [x] No

**Comment by artf** [comment]
Yeah, I think it's just a matter of adding this.toggleVisible() in PanelView.render.
If you want to push a PR, I'd be glad to merge it :)
